coding/pb/pbcodec.go

```diff
@@ -95,8 +95,19 @@ func (c *decoder) Decode(v interface{}) error {
 func ld2pbNode(in *ipld.Node) (*PBNode, error) {
 	n := *in
 	var pbn PBNode
+	var attrs ipld.Node
 
-	if data, hasdata := n["data"]; hasdata {
+	if attrsvalue, hasattrs := n["@attrs"]; hasattrs {
+		var ok bool
+		attrs, ok = attrsvalue.(ipld.Node)
+		if !ok {
+			return nil, errInvalidData
+		}
+	} else {
+		return nil, errInvalidData
```

Comment:
The emptiest object (generated by the reverse function in the same package) I can think of would be:

```js
@attrs: {
  links: [],
  data: ""
}
```

But we may want to be compatible with objects generated from other sources